acceptance/tests/utf8/utf8-in-file-resource.rb

```diff
@@ -0,0 +1,85 @@
+test_name 'utf-8 characters in resource title and param values' do
+
+  confine :except, :platform => [
+    'windows',    # PUP-6983
+    'eos-4',      # PUP-7146
+    'cumulus',    # PUP-7147
+    'cisco_ios',  # PUP-7150
```

Comment:
Only cisco_ios.  This test works when run on cisco-nexus (template: cisco-exr-9k-x86_64).